specification.md

```diff
@@ -440,6 +440,35 @@ an individual job status callback or not. To remove the callback, set it
 to `null`.
 
 
+<a name="jobexecutor-list"></a>
+```java
+List<String> list()
+```
+
+Return a list of job IDs which are known to this executor instance.  The
+returned list MAY contain IDs of jobs which were not submitted via this
+instance, and MAY NOT contain IDs of jobs which have been submitted by this
```

Comment:
```suggestion
instance, and MAY be missing IDs of jobs which have been submitted by this
```

The `MAY NOT` initially read to me like "the list IS NOT ALLOWED to contain IDs of jobs which have been submitted by this instance"